variants/backend-base/config/template.rb

```diff
@@ -17,9 +17,9 @@
 copy_file "variants/backend-base/config/initializers/check_env.rb", "config/initializers/check_env.rb"
 copy_file "variants/backend-base/config/initializers/sentry.rb", "config/initializers/sentry.rb"
 
-gsub_file "config/initializers/filter_parameter_logging.rb", /\[:password\]/ do
-  "%w[password secret session cookie csrf]"
-end
+gsub_file! "config/initializers/filter_parameter_logging.rb",
```

Comment:
note: this relates to #529 - technically I've fixed this gsub so we're now adding to the existing values, but we still want to re-review our options